manta-js/examples/sdk-example/index.ts

```diff
@@ -256,6 +259,76 @@ const multiSbtPostBuild = async (
   return result;
 };
 
+const reserveSbt = async (
+  privateWallet: MantaSbtWallet
+) => {
+  const reserveTx = privateWallet.api.tx.mantaSbt.reserveSbt(null);
+  await publishTransaction([reserveTx]);
+};
+
+const mintSbt = async (
+  privateWallet: MantaSbtWallet
+) => {
+  const assetIdRange: any = await privateWallet.api.query.mantaSbt.reservedIds(polkadotConfig.polkadotAddress);
+  const [startAssetId, endAssetId] = assetIdRange.unwrap();
+  const sbtInfoList: interfaces.SbtInfo[] = [
```

Comment:
Will `reservedIds` return all assetIds without mint?
 
For example, `reserveSbt` will succeed if I call it multiple times? If it succeeds, `reservedIds` will return something like [5, 6, 7] right?

If yes, the value of `sbtInfoList` to return [5, 6, 7] to build the SbtInfo list